docs/overrides.md

```diff
@@ -102,7 +112,18 @@ packages:
               default: "purple"
 ```
 
-In this example, the `helm-overrides-package` Zarf package has a component called `helm-overrides-component` which contains a Helm chart called `podinfo`; note how these names are keys in the `overrides` block. The `podinfo` chart has a `replicaCount` value that is overridden to `2` and a variable called `UI_COLOR` that is overridden to `purple`.
+```yaml
+#values.yaml
+podAnnotations:
+  customAnnotation: "customValue"
+```
+In this example, the `helm-overrides-package` Zarf package has a component called `helm-overrides-component` which contains a Helm chart called `podinfo`; note how these names are keys in the `overrides` block. The `podinfo` chart has a `replicaCount` value that is overridden to `2`, a `podAnnotations` value that is overridden to include `customAnnotation: "customValue"` and a variable called `UI_COLOR` that is overridden to `purple`.
+
+### ValuesFiles
+
+The `valuesFiles` in an `overrides` block are a list of `file`'s. It allows users to override multiple values in a Zarf package component's underlying Helm chart, by providing a file with those values instead of having to include them all indiviually in the `overrides` block.
+
+<b>NOTE:</b> If a value is specified in both a `valuesFile` and as a `value` in the `overrides` block, the value set in the `valuesFile` will take precedence.
```

Comment:
ooo what are your thoughts on this? I would think the `overrides` block would take precedence over the values file?

Reply:
sounds good,  i can be smarter about blindly adding the values to the array as well. As is, the file only takes precedence because it gets added to the values array after the overrides values get added, so on deploy the overrides values get overwritten by the values in the file (if duplicates) when the values map gets populated.